controllers/operands/ssp_test.go

```diff
@@ -137,6 +137,16 @@ var _ = Describe("SSP Operands", func() {
 			Expect(expectedResource.Spec.FeatureGates.DeployVmConsoleProxy).To(BeTrue())
 		})
 
+		It("should create with deployCommonInstancetypes feature gate", func() {
+			hco := commontestutils.NewHco()
+
+			expectedResource, _, err := NewSSP(hco)
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(expectedResource.Spec.FeatureGates.DeployCommonInstancetypes).ToNot(BeNil())
+			Expect(*expectedResource.Spec.FeatureGates.DeployCommonInstancetypes).To(BeTrue())
```

Comment:
nit: You can combine both of them using the `HaveValue` matcher

```suggestion
			Expect(expectedResource.Spec.FeatureGates.DeployCommonInstancetypes).To(HaveValue(BeTrue()))
```